src/nvim/ui.coffee

```diff
@@ -93,14 +93,16 @@ class UI extends EventEmitter
           + '>'
 
     window.addEventListener 'resize', (e) =>
-      clearTimeout @resize_timer if @resize_timer?
-      @resize_timer = setTimeout =>
-        @resize_timer = null
-        col = Math.floor window.innerWidth / @char_width
-        row = Math.floor window.innerHeight / @char_height
-        if col != @total_col or row != @total_row
-          @emit 'resize', col, row
-      , 100
+      if not @resize_timer
+        @resize_timer = setTimeout =>
+          @resize_timer = null
+          col = Math.floor window.innerWidth / @char_width
+          row = Math.floor window.innerHeight / @char_height
+
+          if col != @total_col or row != @total_row
+            @emit 'resize', col, row
+
+        , 250
```

Comment:
mostly arbitrary. I wanted the grid to resize while dragging the window, but don't want to flood neovim. I would welcome a better way to handle it :smile: